cli/create/create.go

```diff
@@ -0,0 +1,82 @@
+package create
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/create/internal/steps"
+	"github.com/tarantool/tt/cli/create/internal/templates"
+	"github.com/tarantool/tt/cli/util"
+	"github.com/tarantool/tt/cli/version"
+)
+
+// FillCtx fills create context.
+func FillCtx(cliOpts config.CliOpts, ctx *cmdcontext.CreateCtx, args []string) error {
```

Comment:
This function never returns error, why not void?

Reply:
Removed error return.